common/src/main/java/org/opensearch/ml/common/transport/upload/MLUploadInput.java

```diff
@@ -74,23 +74,17 @@ public MLUploadInput(FunctionName functionName,
             throw new IllegalArgumentException("model version is null");
         }
         //TODO: enable prebuilt model in 2.6
-//        if (url != null) {
-//            if (modelFormat == null) {
-//                throw new IllegalArgumentException("model format is null");
-//            }
-//            if (modelConfig == null) {
-//                throw new IllegalArgumentException("model config is null");
-//            }
-//        }
+        if (url != null) {
+            if (modelFormat == null) {
```

Comment:
Why do we need to check `modelFormat` two times?

Reply:
Good catch, I will remove this.